filament/backend/include/private/backend/CommandStream.h

```diff
@@ -205,12 +205,12 @@ class NoopCommand : public CommandBase {
 
 // ------------------------------------------------------------------------------------------------
 
-#ifdef NDEBUG
-    #define DEBUG_COMMAND(methodName, params...)
+#if defined(NDEBUG)
+    #define DEBUG_COMMAND(methodName, params, ...)
```

Comment:
I think this should be `#define DEBUG_COMMAND(methodName, ...)` because the `params` parameter is not used in that macro, and the version with a coma essentially removes the first parameter from the list -- which could be confusing.

Reply:
@pixelflinger , makes sense, I made the change